### PR TITLE
Bump version and fix conda-forge update detection

### PR DESCRIPTION
**Pull request by Anthchirp**
Currently the conda-forge bot cannot automatically prepare pull
requests for new versions, as the download URL template does not
work for new versions. This is likely due to incorrect name casing.

I've put in the 'standard' python recipe string, which should
hopefully fix this.

Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by Anthchirp** [comment]
@conda-forge-admin, please rerender

**Comment by Anthchirp** [comment]
This fails with linking error
```
  x86_64-conda-linux-gnu-cc: error: /home/conda/feedstock_root/build_artifacts/uwsgi_1602489933490/_h_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_place/lib/python3.8/config-3.8-x86_64-linux-gnu/libpython3.8.a: No such file or directory
```
on Linux and OSX.

**Comment by ocefpaf** [comment]
The static library was removed from the main python package to create a lighter download. We may need to add `libpython-static` as a dependency here. Ideally, this should not link to `libpython3.8.a` but since it does let's try to add the static dependency and see if we can fix it.

**Comment by Anthchirp** [comment]
is it just a `host:` dependency?

File: recipe/meta.yaml [hunk 1]
```diff
@@ -19,25 +19,26 @@ requirements:
     - {{ compiler('c') }}
     - {{ compiler('cxx') }}
   host:
-    - python
-    - pip
+    - icu
     - jansson
+    - libpython-static
```
Comment (ocefpaf) [inline recipe/meta.yaml:24]: I think you need in host and run.

**Comment by ocefpaf** [comment]
> is it just a `host:` dependency?

I think you'll need as a run time dependency too but maybe now. You can try letting this one finish and if it passes let's merge without it on run.

**Pull request merged**